experiments/long_runs/leaderboard/data_sources.jl

```diff
@@ -0,0 +1,191 @@
+import ClimaAnalysis
+
+"""
+    get_sim_var_dict(diagnostics_folder_path)
+
+Return a dictionary mapping short names to `OutputVar` containing preprocessed
+simulation data. This is used by the function `compute_leaderboard`.
+
+To add a variable for the leaderboard, add a key-value pair to the dictionary
+`sim_var_dict` whose key is the short name of the variable and the value is an
+anonymous function that returns a `OutputVar`. For each variable, any
+preprocessing should be done in the corresponding anonymous function which
+includes unit conversion and shifting the dates.
+
+The variable should have only three dimensions: latitude, longitude, and time.
```

Comment:
I believe the ordering is lon, lat, time? or time lon lat? let's update the docstring to reflect the ordering in the data